### PR TITLE
refactor: updated and decoupled contextService logic (#72)

### PR DESCRIPTION
**Pull request by Souvikns**
* feat: updated and decoupled contextService logic

* feat: creating new context when root context file is missing.

* feat: removed registry & stringify issue

* chore: returning context or undefined in contextAllocator save method

* feat: added warning for saving file.

* feat: lint fixes

**Description**

- This PR fixes the initial issue of context service, where it threw an error when the context file was loaded every time the context file was missing. Now it simply returns `undefined`. 
- This does not break anything and only creates new context service logic is not yet used anywhere. 

**Related issue(s)**

see also #83

### CODE REVIEW

**Comment by Souvikns** [comment]
@magicmatatjahu has once reviewed this, just need the approval to merge.

**Review by derberg — APPROVED** [review]
derberg via Axolo. :shipit:

**Pull request merged**
